chrombpnet/helpers/preprocessing/auto_shift_detect.py

```diff
@@ -35,6 +38,22 @@ def convolve(to_scan, longer_seq):
     for i in range(len(longer_seq) - len(to_scan) + 1):
         vals.append(np.sum(to_scan*longer_seq[i:i+len(to_scan)]))
     return vals
+    
+def filtered_tagalign_stream(tagaligns_stream, genome_file):
```

Comment:
can you add a one liner describing the function here as well

Reply:
Done